src/styles/Footer.css

```diff
@@ -0,0 +1,20 @@
+/* style-for-footer */
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+
+.footer {
+  position: fixed;
+  bottom: 0;
+  width: 100%;
```

Comment:
- [ ] On the details page some parts of the data can't be seen fully because of the footer, kindly modify the styling.

Reply:
fixed position removed